### ISSUE TITLE
Clipboard icon being damaged by the browser

### ISSUE THREAD

**Issue by 1602** [colebemis/feather#212]
The `clipboard` icon sometimes looks like this:

![image](https://user-images.githubusercontent.com/184172/32137433-a760ec26-bc17-11e7-890e-0d011b37147b.png)

I haven't found reliable way to reproduce it, it happens time to time, for example in right top corner here: https://1602.github.io/elm-feather-icons/

I'm not entirely sure whether this is something to do with icon itself or browser or surrounding styles.

**Comment by alexantr** [comment]
What browser, OS?

**Comment by 1602** [comment]
Chrome 61.0.3163.100, Mac OS

**Comment by colebemis** [comment]
@1602 Could you provide some way to replicate this behavior?

**Comment by 1602** [comment]
@colebemis I only reproduced it once, this is "production" case:

1. go to https://1602.github.io/elm-feather-icons/  using mac chrome 61
2. make sure zoom is 100%, observe clipboard icon on the top right corner (looking fine)
3. zoom down to 90%
4. observe damaged clipboard icon on the top

I tried to isolate case here: https://jsfiddle.net/kd5vh9n3/ (also requires changing zoom of the browser). It's not quite the same as "production" case because I purposely set not equal width and height, which isn't the case for "production" but still helps to reproduce the issue (see image at the bottom of this comment). Interesting fact, that this trick only damages one clipboard icon, as I can see.


![image](https://user-images.githubusercontent.com/184172/34079798-d076c5ae-e32c-11e7-89ab-e77c4422f8c6.png)

**Comment by controversial** [comment]
I have the same clipboard rendering issue, even at 100% zoom:
<img width="473" alt="example" src="https://user-images.githubusercontent.com/10377391/57187574-54154600-6ebf-11e9-9d3f-e474bf8ee999.png">


I think I succeeded in creating a [small example](https://jsfiddle.net/luketaylor/3ydLeno4/16/). I believe the issue appears when the icon is positioned with weird fractional pixel values for its size and position. If you open the fiddle in Chrome and resize the output window just a little bit, you should be able to see the issue clearly – the top `path` of the clipboard disappears for certain sizes / positions (which are tied to window size in my example)

**Comment by locness3** [comment]
Looks like that's a problem with chrome, it doesn't happen for me in firefox (on macos)

**Comment by controversial** [comment]
Yeah, I know it is, but given Chrome’s level of penetration and that this issue has affected Chrome for over a year, it would be great if there were a fix.

**Comment by locness3** [comment]
> Yeah, I know it is, but given Chrome’s level of penetration and that this issue has affected Chrome for over a year, it would be great if there were a fix.

True.

**Comment by controversial** [comment]
Applying `scale(.999)` to the icon in CSS seems to fix it, if anyone's looking for a temporary workaround!

**Comment by colebemis** [comment]
Huh, this is such a strange issue. I looked at the SVG file and everything looks normal: https://github.com/feathericons/feather/blob/8f658193d25e943b196a2367319b5d9e85ee9f9f/icons/clipboard.svg#L12-L13

I'm not sure what we can do on our end to fix this. Any ideas?

**Comment by ahtohbi4** [comment]
@colebemis experiment shows to me that replacing `rect` to `path` element in the icon can fix the problem:

*before*

```svg
<svg
  xmlns="http://www.w3.org/2000/svg"
  width="24"
  height="24"
  viewBox="0 0 24 24"
  fill="none"
  stroke="currentColor"
  stroke-width="2"
  stroke-linecap="round"
  stroke-linejoin="round"
>
  <path d="M16 4h2a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H6a2 2 0 0 1-2-2V6a2 2 0 0 1 2-2h2"></path>
  <rect x="8" y="2" width="8" height="4" rx="1" ry="1"></rect>
</svg>
```

*after*

```svg
<svg
  xmlns="http://www.w3.org/2000/svg"
  width="24"
  height="24"
  viewBox="0 0 24 24"
  fill="none"
  stroke="currentColor"
  stroke-width="2"
  stroke-linecap="round"
  stroke-linejoin="round"
>
  <path d="M16 4h2a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H6a2 2 0 0 1-2-2V6a2 2 0 0 1 2-2h2" />
  <path d="M15 2H9a1 1 0 0 0-1 1v2a1 1 0 0 0 1 1h6a1 1 0 0 0 1-1V3a1 1 0 0 0-1-1z" />
</svg>
```

*result (Chrome 75, zoom 90%)*

<img width="240" alt="Снимок экрана 2019-07-08 в 13 04 59" src="https://user-images.githubusercontent.com/3686790/60802450-8c682780-a181-11e9-8232-d9179df58b33.png">

I suppose that the reason is a difference between renders of the elements in Chrome.
